rfcs/0005-managed-attributes-image-automation/README.md

```diff
@@ -0,0 +1,82 @@
+# RFC-0005 Managed attributes on Image Automation
+
+**Status:** provisional
+
+**Creation date:** 2021-12-16
+
+**Last update:** 2021-12-16
+
+## Summary
+
+Image automation controller can update some attributes of a kubernetes object. 
+Today this is limited to image name, image tag and image name+tag. 
+This RFC aims to extend this functionality to other attributes.  
+
+## Motivation
+
+Some automation or observability tools can use label to identify better a 
+kubernetes object. It can be linked to a version, to a date, to a code 
+origin... For multiple reason, the image tag can reflect poorly this
+data. An example can be given by the image reflector controller which 
+can extract a part of the tag and use it to sort and select the correct one.
+
+### Goals
+
+This RFC aims to describe a way to extract such additional value from the
```

Comment:
Maybe change this to a list and add the digest information extraction under the umbrella of this RFC:
- Allow referencing `filterTags` pattern extracted parts from image policies
- Extract digest information and make it available in the markers https://github.com/fluxcd/image-reflector-controller/issues/214